services/app-api/src/emailer/emails/sendQuestionStateEmail.ts

```diff
@@ -43,14 +43,14 @@ export const sendQuestionStateEmail = async (
         packagePrograms
     )
 
-    const packageURL = submissionSummaryURL(
+    const questionResponseURL = submissionQuestionResponseURL(
         contractRev.contract.id,
         config.baseUrl
     )
 
     const data = {
         packageName,
-        submissionURL: packageURL,
+        submissionURL: questionResponseURL,
```

Comment:
Let's change this field to `questionResponseURL` and update the `eta` template to match the CMS version.